TrackingAntennaV3/main/Accelerometer.cpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file   Magnetometer.cpp
+ * @Author Michael Lenover (WARG)
+ * @date   January 25, 2017
+ */
+
+#include "Logger.hpp"
+#include "Accelerometer.hpp"
+#include <Adafruit_LSM303_U.h>
+
+//Create accelerometer object
+static Adafruit_LSM303_Accel_Unified accel;
+
+//Declare accelerometer event variable
+static sensors_event_t accelEvent;
+
+//Initializes accelerometer struct
+volatile Accelerometer accelerometer;
```

Comment:
In the Accelorometer header file, add `extern volatile Accelerometer accelerometer;`. Doing this exposes the variable to other files (right now they cant). You can read up on  what extern does with a google search